### ISSUE TITLE
Cut - Paste nuances

### ISSUE THREAD

**Issue by vitaaliy** [aaFn/Bookmark-search-plus-2#15]
In native FF mode when I chose some bookmark and _Cut_ it - visually happens nothing. And if I fail to move it somewhere - it will remain where it was. Moving is done only after my _Paste_ command. In _Bookmark-search-plus-2_ after _Cut_ click chosen bookmark disappears right away. I tried to move it to some folder - but nothing happened. Yes. But after several seconds the chosen bookmark returns to its initial position. Luckily it was not deleted forever!

**Comment by aaFn** [comment]
Hello @vitaaliy, I hadn't paid attention, thank you for the observation .. and indeed, when cutting, the object even becomes lighter in color to mark that it is under a cut.
Ok, I'll see to change that.

On the second effect, strange, it is working for me. Maybe the big numbers are playing a bad thing again ? Let me first adjust the behavior, and then we should investigate if the problem continues to happen, and I should debug ..

For now I am setting this as enhancement, and we'll open another one later if there's a bug remaining.
Thank you.

**Comment by aaFn** [comment]
@vitaaliy, implemented in 2.0.5, let's close that issue if you agree.

And if you still have the second problem of "cut" and "paste" not being effective, let's open a new issue to track that specific one.

**Comment by aaFn** [comment]
Closing as resolved.
